docs/result-v1-0.txt

```diff
@@ -0,0 +1,725 @@
+
+Running: calrissian --max-ram 8G --max-cores 4 --default-container debian:stretch-slim --outdir=/output/tmpsb0vnoam --quiet v1.0/v1.0/cat-from-dir.cwl v1.0/v1.0/cat-from-dir-job.yaml
+Test [190/197] stdin_from_directory_literal_with_literal_file: Pipe to stdin from literal File via a Directory literal
+
+Running: calrissian --max-ram 8G --max-cores 4 --default-container debian:stretch-slim --outdir=/output/tmpoe_2o_3p --quiet v1.0/v1.0/cat-from-dir.cwl v1.0/v1.0/cat-from-dir-with-literal-file.yaml
+Test [191/197] directory_literal_with_literal_file_nostdin: Test non-stdin reference to literal File via a Directory literal
+
+Running: calrissian --max-ram 8G --max-cores 4 --default-container debian:stretch-slim --outdir=/output/tmp7ivpjm48 --quiet v1.0/v1.0/cat3-from-dir.cwl v1.0/v1.0/cat-from-dir-with-literal-file.yaml
+Test [192/197] no_inputs_commandlinetool: Test CommandLineTool without inputs
+
+Running: calrissian --max-ram 8G --max-cores 4 --default-container debian:stretch-slim --outdir=/output/tmptaln_li3 --quiet v1.0/v1.0/no-inputs-tool.cwl
+Test [193/197] no_outputs_commandlinetool: Test CommandLineTool without outputs
+
+Running: calrissian --max-ram 8G --max-cores 4 --default-container debian:stretch-slim --outdir=/output/tmppr8vackp --quiet v1.0/v1.0/no-outputs-tool.cwl v1.0/v1.0/cat-job.json
+Test [194/197] no_inputs_workflow: Test Workflow without inputs
+
+Running: calrissian --max-ram 8G --max-cores 4 --default-container debian:stretch-slim --outdir=/output/tmp1ocyp1w7 --quiet v1.0/v1.0/no-inputs-wf.cwl
+Test [195/197] no_outputs_workflow: Test Workflow without outputs
+
+Running: calrissian --max-ram 8G --max-cores 4 --default-container debian:stretch-slim --outdir=/output/tmpsn5kk2db --quiet v1.0/v1.0/no-outputs-wf.cwl v1.0/v1.0/cat-job.json
+Test [196/197] anonymous_enum_in_array: Test an anonymous enum inside an array inside a record
+
+Running: calrissian --max-ram 8G --max-cores 4 --default-container debian:stretch-slim --outdir=/output/tmpnn_106ud --quiet v1.0/v1.0/anon_enum_inside_array.cwl v1.0/v1.0/anon_enum_inside_array.yml
+Test [197/197] schema-def_anonymous_enum_in_array: Test an anonymous enum inside an array inside a record, SchemaDefRequirement
+
+Running: calrissian --max-ram 8G --max-cores 4 --default-container debian:stretch-slim --outdir=/output/tmpkoau27hd --quiet v1.0/v1.0/anon_enum_inside_array_inside_schemadef.cwl v1.0/v1.0/anon_enum_inside_array_inside_schemadef.yml
+Traceback (most recent call last):
+  File "/usr/local/bin/cwltest", line 8, in <module>
+    sys.exit(main())
+  File "/usr/local/lib/python3.10/site-packages/cwltest/main.py", line 236, in main
+    utils.generate_badges(args.badgedir, ntotal, npassed)
+  File "/usr/local/lib/python3.10/site-packages/cwltest/utils.py", line 132, in generate_badges
+    os.mkdir(badgedir)
+FileExistsError: [Errno 17] File exists: '/output/badges-1.0.2'
```

Comment:
whoops :-)

Reply:
will re-run. 
@mr-c with 1.2.1_proposed, I get `required` below 100%, something I will have to look into.
any clues?

Reply:
Here are the new mandatory tests in CWL v1.2.1:

For workflows: https://deploy-preview-115--cwl-v1-2-dev.netlify.app/workflow#New_Mandatory_Conformance_tests_for_v1.2.1 (just 1)
For CommandLineTools: https://deploy-preview-115--cwl-v1-2-dev.netlify.app/commandlinetool#New_Mandatory_Conformance_Tests_for_v1.2.1 (21 news mandatory tests)

Reply:
Looks like the failing tests are
- params_broken_null [log](https://github.com/Duke-GCB/calrissian/pull/164/files#diff-83f52c3e68f7660d96d2f8f4fafcd728e7cfe76fc6b2a4d7737ba460fefa6b62R1933-R1958)
- length_for_non_array [log](https://github.com/Duke-GCB/calrissian/pull/164/files#diff-83f52c3e68f7660d96d2f8f4fafcd728e7cfe76fc6b2a4d7737ba460fefa6b62R1959-R1989)
- colon_in_paths [log of test timeout](https://github.com/Duke-GCB/calrissian/pull/164/files#diff-83f52c3e68f7660d96d2f8f4fafcd728e7cfe76fc6b2a4d7737ba460fefa6b62R1996-R2000)
- filename_with_hash_mark [log](https://github.com/Duke-GCB/calrissian/pull/164/files#diff-83f52c3e68f7660d96d2f8f4fafcd728e7cfe76fc6b2a4d7737ba460fefa6b62R2037-R2041)
- capture_files [log](https://github.com/Duke-GCB/calrissian/pull/164/files#diff-83f52c3e68f7660d96d2f8f4fafcd728e7cfe76fc6b2a4d7737ba460fefa6b62R2045-R2082)
- capture_dirs [log](https://github.com/Duke-GCB/calrissian/pull/164/files#diff-83f52c3e68f7660d96d2f8f4fafcd728e7cfe76fc6b2a4d7737ba460fefa6b62R2083-R2120)
- paramref_arguments_runtime: [log](https://github.com/Duke-GCB/calrissian/pull/164/files#diff-83f52c3e68f7660d96d2f8f4fafcd728e7cfe76fc6b2a4d7737ba460fefa6b62R2136-R2155)